language/de/common.php

```diff
@@ -526,7 +526,7 @@
 	'NO_POSTS_TIME_FRAME'		=> 'Für den ausgewählten Zeitraum existieren keine Beiträge in diesem Thema.',
 	'NO_FEED_ENABLED'			=> 'Auf diesem Board sind keine Feeds verfügbar.',
 	'NO_FEED'					=> 'Der angeforderte Feed ist nicht verfügbar.',
-	'NO_STYLE_DATA'				=> 'Die Style-Daten konnten nicht abgerufen werden',
+	'NO_STYLE_DATA'				=> 'Die Style-Daten für user_style %s, die für user_id %s gesetzt sind, konnten nicht abgerufen werden',
```

Comment:
Siehe Kommentar in #394

Reply:
Kannst du einfach von meiner Änderung kopieren...